virttest/utils_test/qemu.py

```diff
@@ -1774,9 +1774,49 @@ def check_memory(self, vm=None):
         self.os_type = self.params.get("os_type")
         guest_mem_size = super(MemoryHotplugTest, self).get_guest_total_mem(vm)
         vm_mem_size = self.get_vm_mem(vm)
-        threshold = vm_mem_size * 0.06
+        default_threshold = vm_mem_size * 0.06
+        threshold = float(self.params.get("threshold", default_threshold))
         if abs(guest_mem_size - vm_mem_size) > threshold:
```

Comment:
My test result: smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.spice.qcow2.virtio_scsi.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.hotplug_memory_repeat.backend_file.policy_default: FAIL (59.25 s)

The attachment file is it's debug.log.